mars/tensor/execution/linalg.py

```diff
@@ -72,24 +72,25 @@ def _solve_triangular(ctx, chunk):
             import scipy.linalg
 
             ctx[chunk.key] = scipy.linalg.solve_triangular(a, b, lower=chunk.op.lower)
-        else:
+        elif is_sparse_module(xp):
```

Comment:
Cupy now supports solve_triangular, refer to https://docs-cupy.chainer.org/en/stable/reference/generated/cupyx.scipy.linalg.solve_triangular.html#cupyx.scipy.linalg.solve_triangular

No else any more.